datafusion/sqllogictest/test_files/insert.slt

```diff
@@ -296,8 +296,11 @@ insert into table_without_values(field1) values(3);
 1
 
 # insert NULL values for the missing column (field1), but column is non-nullable
-statement error Execution error: Invalid batch column at '0' has null but schema specifies non-nullable
+statement error
 insert into table_without_values(field2) values(300);
+----
```

Comment:
This is due to we have checking in insert into plan now, before this we only have the check for execution plan check.